apps/scale-count.html.markerb

```diff
@@ -324,4 +324,4 @@ If a Machine is misbehaving (for instance, it's not `stop`ping successfully), yo
 fly machine destroy --force 0e286039f42e86
 ```
 
-<div class="callout"> If you `destroy` a Machine, any volume that it had been using still exists until you either `fly volumes delete` it or `fly destroy` the Fly App it belongs to.</div>
+<div class="callout"> If you `destroy` a Machine, any volume that it had been using still exists until you either `fly volumes delete` it or `fly apps destroy` the Fly App it belongs to.</div>
```

Comment:
My preference would be not to use commands as verbs like this. In this case, remove the backticks from "If you destroy a Machine" and rewrite the rest like "until you either delete the volume with the `fly volumes delete` command or delete the app the volume belongs to with the the `fly apps destroy` command."

Reply:
Oh no!

Reply:
I tidied the callout. My reasoning here is that it's unnecessarily long to include all the commands.